src/OrderBookHandlers.ts

```diff
@@ -27,20 +27,23 @@ OrderBookContract.MarketCreateEvent.handler(({ event, context }) => {
   });
 });
 
-OrderBookContract.OrderChangeEvent.loader(({ event, context }) => {});
+OrderBookContract.OrderChangeEvent.loader(({ event, context }) => { });
 
 OrderBookContract.OrderChangeEvent.handler(({ event, context }) => {
   const eventOrder = event.data.order;
   const timestamp = tai64ToDate(event.data.timestamp);
   const order: spotOrderEntity | null = eventOrder
     ? {
-        id: eventOrder.id,
-        trader: eventOrder.trader.value,
-        base_token: eventOrder.base_token.value,
-        base_size: decodeI64(eventOrder.base_size),
-        base_price: eventOrder.base_price,
-        timestamp,
-      }
+      id: eventOrder.id,
+      trader: eventOrder.trader.value,
+      base_token: eventOrder.base_token.value,
+      base_size: decodeI64(eventOrder.base_size),
+      order_type: eventOrder.base_size.value.toString() == "0"
```

Comment:
Suggestion: You can do the check without converting to string

```suggestion
      order_type: eventOrder.base_size.value === 0n
```